src/main/java/pl/sda/library/infrastructure/jpa/AuthorDto.java

```diff
@@ -24,11 +23,19 @@
     @Column(name = "last_name")
     private String lastName;
 
-    public Author toModel() {
+    Author toModel() {
         Author author = new Author();
         author.setId(getId());
         author.setFirstName(getFirstName());
         author.setLastName(getLastName());
         return author;
     }
+
+    static AuthorDto fromDomain(Author author) {
```

Comment:
zróbmy spójnie

albo para: toModel/fromModel, albo toDomain/fromDomain (osobiście preferuje to drugie)

Reply:
niech będzie domain